src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,102 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serving multiple applications including James. 
+The keyspace of each James server need to be protected from another, 
+that also means, by the credential provided for a James server, 
+I cannot use it to access or modify other keyspaces. 
+
+We recommend you to not use the default Cassandra credential and provide 
+a different one with a minimized permissions for each application. 
+
+### Prerequisites
+
+We're gonna use the super user to create roles and grant permissions for them. 
```

Comment:
is there a single super user for a given server?

Reply:
do you mean s/super user/super users?

Reply:
I don't know, it depends on the answer (: